addrmgr/error_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2024 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package addrmgr
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestErrors(t *testing.T) {
+	tests := []struct {
```

Comment:
The reason this test has a tests struct but only one test case is because I'm planning to add more error types in a future PR, but didn't want to include that change in this PR.